RELEASE.md

```diff
@@ -283,6 +289,8 @@ You can add this label before or after the PR is merged. Grafanabot will open a
 branch with the merge commit of the PR you labelled. See [PR#2290 (original PR)](https://github.com/grafana/mimir/pull/2290) and
 [PR#2364 (backport PR)](https://github.com/grafana/mimir/pull/2364) for an example pair.
 
+In the case of backport is failed. Follow step from grafanabot's comment to do a manual cherry-pick.
```

Comment:
```suggestion
In case the automated backport fails, follow the steps from grafanabot's comment to do a manual cherry-pick.
```

Reply:
Not grammar expert. Wouldn't this be more correct?
```suggestion
In case the automated backport failed, follow the steps from grafanabot's comment to do a manual cherry-pick.```

Reply:
sounds ok, just lose the extra back ticks